lib/crypto/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "crypto"
+name = "openzeppelin-crypto-rs"
```

Comment:
Isn't it contradicting to the [official guidelines](https://rust-lang.github.io/api-guidelines/naming.html)?

> Crate names should not use -rs or -rust as a suffix or prefix. Every crate is Rust! It serves no purpose to remind users of this constantly.

Reply:
Of course there are many crates that doesn't follow that convention. But seems it is an official recomendation

Reply:
Yes! I'll remove the suffix, was thinking something else. thx

Reply:
I am not a fun of this suffix; I vote for just  `openzeppelin-crypto`.

Reply:
`openzeppelin-crypto` yeah. This one seems good also for me